src/pages/Index.vue

```diff
@@ -12,7 +12,9 @@
       <h2 class="text-white text-center text-2xl max-w-sm mb-4">
         We dream, scheme, and support each other in our careers
       </h2>
-      <BaseButton @click="showDialog = true" class="bg-gray-100">Stay Updated</BaseButton>
+      <BaseButton @click="showDialog = true" class="bg-gray-100 font-serif"
```

Comment:
This should be `font-sans`
```suggestion
      <BaseButton @click="showDialog = true" class="bg-gray-100 font-sans"
```